projects/plugins/protect/src/js/components/footer/index.jsx

```diff
@@ -21,51 +27,78 @@ const ProductPromotion = () => {
 		run
 	);
 
+	// TODO: Update with new paid Protect product
 	const { securityBundle } = useProtectData();
 	const { hasRequiredPlan } = securityBundle;
 
+	// TODO: Update button links with appropriate Cloud links
 	if ( hasRequiredPlan ) {
-		const getStartedUrl = getRedirectUrl( 'protect-footer-get-started-scan' );
-
 		return (
 			<div className={ styles[ 'product-section' ] }>
-				<IconsCard products={ [ 'backup', 'scan', 'anti-spam' ] } />
-				<Title>
-					{ __( 'Learn how Jetpack Scan increases your site protection', 'jetpack-protect' ) }
-				</Title>
+				<Title>{ __( 'Get access to our Cloud', 'jetpack-protect' ) }</Title>
 				<Text mb={ 3 }>
 					{ __(
-						'With your Jetpack Security bundle you have access to Jetpack Scan. Automatically scan your site from the Cloud, get email notifications and perform one-click fixes.',
+						'With your Protect upgrade, you have free access to scan your site on our Cloud, so you can be aware and fix your threats even if your site goes down. ',
 						'jetpack-protect'
 					) }
+					<Button variant="link" weight="regular" href="#">
+						{ __( 'Learn more', 'jetpack-protect' ) }
+					</Button>
 				</Text>
 
-				<Button variant="link" isExternalLink={ true } weight="regular" href={ getStartedUrl }>
-					{ __( 'Get Started', 'jetpack-protect' ) }
+				<Button variant="secondary" weight="regular" href="#">
```

Comment:
For the "Go to Cloud" button, could we use the same URL that the sidebar currently uses to direct the user to their Scan page on Jetpack Cloud?

<img width="192" alt="Screen Shot 2022-10-04 at 12 37 00 PM" src="https://user-images.githubusercontent.com/10933065/193900909-502556d7-e147-4390-b14a-307dbf9773ef.png">